mne/viz/misc.py

```diff
@@ -586,6 +586,8 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
         Color to plot with each dipole. If None default colors are used.
     show : bool
         Show figure if True.
+    linewidth : int
+        Line width in 2D plot.
```

Comment:
please revert this. You can fix this after plotting by working with the outputed fig object.